rush.json

```diff
@@ -411,6 +411,13 @@
       "shouldPublish": true,
       "versionPolicyName": "prerelease-monorepo-lockStep"
     },
+    {
+      "packageName": "@itwin/map-layers-auth",
+      "projectFolder": "extensions/map-layers-auth",
+      "reviewCategory": "extensions",
```

Comment:
@calebmshafer should this and map-layers be changed from "extensions" to "frontend"? They are not extensions.

Reply:
yes, I agree it should switch to frontend.

Technically we should move this out of an "extensions" folder.

Reply:
@pmconne ok to move it in a separate PR ?

Reply:
Yes, that seems preferable.